templates/frame/index.html

```diff
@@ -0,0 +1,452 @@
+                                height="140",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-image-container__image",}) | safe
+                }}
+              </div>
+              <hr class="p-rule--muted" />
+            </div>
+            <div class="p-equal-height-row__item">
+              <h3 class="p-heading--5">Intel NUC</h3>
+              <div class="u-hide--large p-cta-block">
+                <a href="/download/intel-nuc">Install Ubuntu Frame on Intel NUC&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
+          <div class="p-equal-height-row__col is-borderless">
+            <div class="p-equal-height-row__item">
+              <div class="p-image-container">
+                {{ image(url="https://assets.ubuntu.com/v1/ef894f14-dragonboard.png",
+                                alt="",
+                                width="852",
+                                height="569",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+              <hr class="p-rule--muted" />
+            </div>
+            <div class="p-equal-height-row__item">
+              <h3 class="p-heading--5">Dragonboard</h3>
+              <div class="u-hide--large p-cta-block">
+                <a href="/download/qualcomm-dragonboard-410c">Install Ubuntu Frame on Dragonboard&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <hr class="p-rule--muted u-hide--small u-hide--medium" />
+      </div>
+      <div class="grid-row u-hide--small u-hide--medium">
+        <div class="grid-col-2 grid-col-start-large-3">
+          <a href="/download/raspberry-pi-core">Install Ubuntu Frame on Raspberry Pi&nbsp;&rsaquo;</a>
+        </div>
+        <div class="grid-col-2">
+          <a href="/download/intel-nuc">Install Ubuntu Frame on Intel NUC&nbsp;&rsaquo;</a>
+        </div>
+        <div class="grid-col-2">
+          <a href="/download/qualcomm-dragonboard-410c">Install Ubuntu Frame on Dragonboard&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-7 col-6">
+        <div class="p-cta-block">
+          <a href="https://discourse.ubuntu.com/t/where-does-ubuntu-frame-work/23193"
+             aria-label="Learn more information about Ubuntu Frame graphic support">More information about Ubuntu Frame graphic support&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="p-rule is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="row">
+      <h2>
+        Need help? Let’s get your device faster to market
+        <br />
+        <a href="/internet-of-things/smart-displays#get-in-touch"
```

Comment:
```suggestion
        <a href="/contact-us"
```
Considering this page doesn't have a dedicated static form, the url should point to root static form. Unless it's a specific requirement to point to to smart-displays?

Reply:
Yes, it is a requirement to point to the smart displays page [here](https://docs.google.com/document/d/1P4xOahtq6VVY6llAAAmQ4cX7ip4zzNXxXciuZ9joxMw/edit?disco=AAABlDklrM0)
It's the same as in production and now and they want it left the same way